realm/realm-library/src/androidTest/java/io/realm/internal/JNICloseTest.java

```diff
@@ -18,86 +18,14 @@
 
 import android.test.AndroidTestCase;
 
-import java.io.Closeable;
-import java.io.File;
-import java.util.ArrayList;
-import java.util.List;
-
 import io.realm.RealmFieldType;
 import io.realm.TestHelper;
 
+
 // Tables get detached
 
 public class JNICloseTest extends AndroidTestCase {
 
```

Comment:
Not need anymore since `Table.close()` is removed. (It is removed since no-one is using it and it should be handled by Phantom Ref in the quite near future.)